### PR TITLE
New additional stats in Stats for Nerds

### PR DESCRIPTION
**Pull request by httpsnet**
Adds Ascension Speed Multipliers and Golden Quark Multipliers to Stats for Nerds.
Changed calculateAscensionAcceleration() for adding Ascension Speed Multipliers and added calculateAscensionSpeedMultiplier().
Changed calculateGoldenQuarkGain() for adding Golden Quark Multipliers and added calculateGoldenQuarkMultiplier().
Added Stats for Nerds are arranged according to the order of appearance in the Synergism Stage.

Fixed incorrect calculations due to merging Golden Quarks I and Cookie Upgrade 19 calculations. This fix increases Golden Quark.
Fixed Golden Revolution II cap not calculating +100%.
Fixed the display of the Industrial Daily Codes perk to Sing 200. This is because the actual effect was Sing 200.
Added s100 to Synergism Stage.

### CODE REVIEW

File: src/Calculate.ts [hunk 1]
```diff
@@ -1527,29 +1536,32 @@ export const calculateQuarkMultiplier = () => {
...
-        1 + Math.max(0, Math.log10(player.challenge15Exponent + 1) - 20) / 2,
-        1 + player.worlds.BONUS / 100,
-        (+player.singularityUpgrades.goldenQuarks1.getEffect().bonus) *
-        1 + 0.12 * player.cubeUpgrades[69],
-        1 + calculateEventBuff('Golden Quarks'),
-        fastForwardMultiplier,
-        (player.highestSingularityCount >= 100) ? 1 + player.highestSingularityCount / 250 : 1
-    ]);
+    return {list: arr,
+        mult: productContents(arr) + bonus}
```
Comment (KhafraDev) [inline src/Calculate.ts:1560]:
```suggestion
    return {
        list: arr,
        mult: productContents(arr) + bonus
    }
```

File: src/Calculate.ts [hunk 2]
```diff
@@ -1418,33 +1418,42 @@ export const calculateTimeAcceleration = () => {
...
     const exponent = (player.singularityUpgrades.singAscensionSpeed.level > 0) ?
         ((baseMultiplier >= 1) ?
             1.03:
             0.97) :
         1;
+    arr[16] = Math.pow(baseMultiplier, exponent) / baseMultiplier;
 
-    return Math.pow(baseMultiplier, exponent)
+    return {list: arr,
+        mult: productContents(arr)}
```
Comment (KhafraDev) [inline src/Calculate.ts:1452]:
```suggestion
    return {
        list: arr,
        mult: productContents(arr)
    }
```

**Review by KhafraDev — COMMENTED** [review]
just some style changes other lgtm

**Review by KhafraDev — APPROVED** [review]

**Comment by httpsnet** [comment]
Immaculate Alchemy calculations are currently incorrect, please wait for a fix

**Comment by httpsnet** [comment]
Fixed. I confirmed again that the calculation was correct

**Comment by httpsnet** [comment]
Added No Singularity Upgrades to Quark Multiplier Stats
Support new Progression Chat and Questions child thread names
Fixed one mistake in #417

**Comment by httpsnet** [comment]
I checked the code included in the PR again.
I didn't find any issues this time, but I've done this many times in the past, making it harder to find.


Purchasing Platonic ALPHA...? now updates the Platonic Alpha background color.

Changed Automation Platonic Upgrades - Level 1 and Automation Cube Upgrades - Level 1 to use Singularity Challenge.
Removed one eslint-disable-line that is not needed.

Updated the Synergism Stage. Since I created the Synergism Stage, I will continue to support it.

Fixed a bug where Hepteracts toggle was required for Automation Open Cubes to work. This ended up being #417 with a copy of the if that I didn't need.

Fixed an issue where Talisman Shards could be owned over 1e300.
4D-7D cubes capped at 1e300 so they can no longer be owned beyond 1e300. This is one of the problems theoretically caused by repeated updates.

**Pull request merged**